src/Microsoft.Extensions.FileProviders.Physical/Internal/FileSystemInfoHelper.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Extensions.FileProviders.Physical.Internal
+{
+    /// <summary>
+    /// Helpful methods for FileSystem APIs.
+    /// </summary>
+    public static class FileSystemInfoHelper
```

Comment:
Yeah that would be good... less future pain for us. Can you send an update for that? (New PR of course)